examples/generative/dcgan_overriding_train_step.py

```diff
@@ -154,10 +156,10 @@ def train_step(self, real_images):
         )
 
         # Sample random points in the latent space
-        random_latent_vectors = tf.random.normal(shape=(batch_size, self.latent_dim))
+        random_latent_vectors = keras.random.normal(shape=(batch_size, self.latent_dim))
```

Comment:
Critically, this needs to be seeded with a `SeedGenerator` instance attached to the model.

Reply:
Sure, made the required changes